accountClient/accountClient.go

```diff
@@ -20,19 +22,19 @@ type E3dbAccountClient struct {
 }
 
 // CreateAccount attempts to create an e3db account using the provided params, returning created account and error (if any).
-func (c *E3dbAccountClient) CreateAccount(ctx context.Context, params CreateAccountRequest) (*CreateAccountResponse, *e3dbClients.RequestError) {
+func (c *E3dbAccountClient) CreateAccount(ctx context.Context, params CreateAccountRequest) (*CreateAccountResponse, error) {
```

Comment:
I don't think you can make this change without breaking any code that tries to do things like `https://github.com/tozny/metrics-service/commit/a16d7ee99700c75dd4a66bf5daa95267a01766f6#diff-c47669012d7664f9cbee69c67bf8bedfR90` so make sure you test this change out for those cases